app/components/authentication_tokens/details/view.rb

```diff
@@ -3,7 +3,10 @@
 module AuthenticationTokens
   module Details
     class View < ViewComponent::Base
-      attr_reader :token
+      include Pundit::Authorization
+      include UsersHelper
+
+      attr_reader :token, :current_user
```

Comment:
`current_user` will be `nil` as it is not passed in the component in `initialize`

```
def initialize(token, current_user = Current.user)
  @token             = token
  @current_user = current_user
end
```